src/pages/Login/index.js

```diff
@@ -1,44 +1,3 @@
-/* eslint-disable max-len */
-import React from 'react';
-import { Redirect } from 'react-router-dom';
-import { useLoginSelector, useLogintDispatch } from '../../hooks/login';
-import './style.css';
-import logo from '../../static/codelibrarylogo.png';
+import Login from '../../components/Login';
 
-export default function Login() {
-    const { accessToken } = useLoginSelector();
-    const { signInUser } = useLogintDispatch();
-
-    const handleClick = () => {
-        signInUser();
-    };
-
-    return (
-        <>
-            {accessToken ? (
-                <Redirect to="/" />
-            ) : (
-                <div className="Login">
-                    <div>
-                        <div className="loginlogo">
-                            <img src={logo} alt="code library logo" />
-                        </div>
-                        <div
-                            id="customBtn"
-                            className="customGPlusSignIn"
-                            onClick={handleClick}
-                            onKeyDown={handleClick}
-                            role="button"
-                            tabIndex={0}
-                        >
-                            <span className="icon" />
-                            <span className="buttonText">
-                                Sign in with Google
-                            </span>
-                        </div>
-                    </div>
-                </div>
-            )}
-        </>
-    );
-}
```

Comment:
Importing all of the code from the login component is not what I meant by moving the code. This Login page component should get supply all of the state and event handlers to the login component